src/runtime/ExecutionState.cpp

```diff
@@ -62,6 +63,31 @@ FunctionObject* ExecutionState::resolveCallee()
     return nullptr;
 }
 
+Script* ExecutionState::resolveOuterScript()
```

Comment:
How about use Optional<Script*> instead of Script*?

Reply:
Thanks for the review. I fixed it :)